### PR TITLE
Feature/ad 41 signup & edit first name and last name added

### PR DESCRIPTION
**Pull request by AlexW555**
- Added first name and last name fields to edit account page and registration pages.
- Additionally added bio to edit so that the user can add more details to their page once they are signed up

Screenshot 2021-08-24 at 9 57 30 am

### CODE REVIEW

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -5,9 +5,21 @@
...
+      <%= form.text_field :first_name, autofocus: true, autocomplete: 'email', required: true %>
+    </div>
+    <div class="field">
+      <%= form.label :last_name %><br>
+      <%= form.text_field :last_name, autofocus: true, autocomplete: 'email', required: true %>
+    </div>
+    <div class="field">
+      <%= form.label :bio %><br>
+      <%= form.text_field :bio, autofocus: true, autocomplete: 'bio' %>
+    </div>
```
Comment (garethdandrews) [inline app/views/devise/registrations/edit.html.erb:19]: Could you make this a text area?

**Comment by garethdandrews** [comment]
Now we have multiple fields on sign up / edit registration, perhaps its worth adding some asterisks or something to indicate that a field is required.
I think avatar could use a label too.

**Comment by AlexW555** [comment]
Screenshot 2021-08-25 at 11 17 57 am

**Review by garethdandrews — APPROVED** [review]
LGTM 👍

**Pull request merged**